test/jdk/javax/swing/JRadioButton/8033699/bug8033699.java

```diff
@@ -168,11 +166,14 @@ private static void createAndShowGUI() {
         radioBtnSingle = new JRadioButton("Not Grouped");
         radioBtnSingle.setSelected(true);
 
-        mainFrame.getContentPane().add(btnStart);
-        mainFrame.getContentPane().add(box);
-        mainFrame.getContentPane().add(radioBtnSingle);
-        mainFrame.getContentPane().add(btnEnd);
+        // Create main vertical box
+        Box mainBox = Box.createVerticalBox();
+        mainBox.add(btnStart);
+        mainBox.add(box);
+        mainBox.add(radioBtnSingle);
+        mainBox.add(btnEnd);
 
+        mainFrame.getContentPane().add(mainBox);
```

Comment:
```suggestion
        mainFrame.add(mainBox);
```
At this point, @honkar-jdk's can be applied. I'll leave it to your discretion, either way is fine with me.